build/Publish.fs

```diff
@@ -35,19 +35,22 @@ type PublishResult =
         | otherwise -> false
 
 let publishSdk (projectDir: string) (version: string) (nugetApiKey: string) : PublishResult =
-    let buildNugetCmd = String.concat " " [
-        "build"
+    let cmd (name: string) = String.concat " " [
+        name
         "--configuration Release"
         $"-p:Version={version}"
     ]
-    
-    if Shell.Exec("dotnet", buildNugetCmd, projectDir) <> 0 then
-        PublishResult.Failed(projectDir, "failed to build the nuget package")
+
+    if Shell.Exec("dotnet", cmd "build", projectDir) <> 0 then
+        PublishResult.Failed(projectDir, $"failed to build the project at {projectDir}")
     else
+        if Shell.Exec("dotnet", cmd "pack", projectDir) <> 0 then
+            PublishResult.Failed(projectDir, $"failed to pack the project at {projectDir}")
+        else
         let releaseDir = Path.Combine(projectDir, "bin", "Release")
```

Comment:
Should this line and the lines below be indented?

Reply:
`else` branch usually doesn't _have_ to be indented in F# (to reduce nesting)